app/Repositories/LocationRepository.php

```diff
@@ -247,4 +247,12 @@ public function getTotalUsersInDay($location, $filter)
 
         return $data;
     }
+
+    public function getLocationIdByWorkspace($workspace_id)
+    {
+        return $this->model
+            ->where('workspace_id', $workspace_id)
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/LocationRepository.php</br>**Line**: 254</br>Variable "workspace_id" is not in valid camel caps format